themes/gfsc/layouts/shortcodes/anchorfm.html

```diff
@@ -1,3 +1,3 @@
 <iframe src="https://anchor.fm/gfscstudio/embed/episodes/{{ .Get 0}}" height="100%" width="100%" frameborder="0" scrolling="no"></iframe>
 
-{{ "*Subscribe via [RSS](https://anchor.fm/s/a948e35c/podcast/rss), [Sticher](https://www.stitcher.com/show/geeks-for-social-change), [Spotify](https://open.spotify.com/show/5W8FIdNNr3i1jPj02uUevO), [Radio Public](https://radiopublic.com/geeks-for-social-change-60Xbpb), [Apple Podcasts](https://podcasts.apple.com/us/podcast/geeks-for-social-change/id1639344512), [Amazon Music](https://music.amazon.co.uk/podcasts/e37fd22d-7c11-4cc9-a282-951cc205733c/geeks-for-social-change) or [Anchor FM](https://anchor.fm/gfscstudio/)*" | markdownify }}
+{{ "*Subscribe via [RSS](https://anchor.fm/s/a948e35c/podcast/rss), [Sticher](https://www.stitcher.com/show/geeks-for-social-change), [Spotify](https://open.spotify.com/show/5W8FIdNNr3i1jPj02uUevO), [Radio Public](https://radiopublic.com/geeks-for-social-change-60Xbpb), [Apple Podcasts](https://podcasts.apple.com/us/podcast/geeks-for-social-change/id1639344512), [Amazon Music](https://music.amazon.co.uk/podcasts/e37fd22d-7c11-4cc9-a282-951cc205733c/geeks-for-social-change) or [Spotify for Podcasters](https://anchor.fm/gfscstudio/)*" | markdownify }}
```

Comment:
```suggestion
{{ "*Subscribe via [RSS](https://anchor.fm/s/a948e35c/podcast/rss), [Sticher](https://www.stitcher.com/show/geeks-for-social-change), [Spotify](https://open.spotify.com/show/5W8FIdNNr3i1jPj02uUevO), [Radio Public](https://radiopublic.com/geeks-for-social-change-60Xbpb), [Apple Podcasts](https://podcasts.apple.com/us/podcast/geeks-for-social-change/id1639344512), [Amazon Music](https://music.amazon.co.uk/podcasts/e37fd22d-7c11-4cc9-a282-951cc205733c/geeks-for-social-change) or [Spotify for Podcasters](https://podcasters.spotify.com/pod/show/gfscstudio/)*" | markdownify }}
```

lets update the link while we're here so we don't rely on there redirect forever